spec/support/select2_spec_helper.rb

```diff
@@ -0,0 +1,10 @@
+module Select2SpecHelper
+  def select_user(user)
+    find(".select2-selection").click
+    find(:xpath, "//body").find(".select2-search input.select2-search__field").set(user.full_name)
+    sleep(0.5)
```

Comment:
Is this absolutely necessary ? this is usually suspicious because it means we're not using capybara correctly, it should wait for the events on its own. this is usually the source for flaky tests 😨